src/generator/css/base.less

```diff
@@ -1025,3 +1573,425 @@ body.post-page {
         }
     }    
 }
+
+.main-banner-block {
+    @apply flex relative items-center justify-center w-full min-h-80;
+}
+
+.main-banner-text {
+    @apply z-10 text-center min-h-72 bg-[#0073D4] w-2/4;	
+
+    clip-path: polygon(0% 0%,90% 0%,100% 100%,10% 100%);	
+}
+
+.inclined-contour  {
+  
+}
+
+.blog-recent {
+    @apply bg-[var(--color-accent-dark)] pt-20 sm:pt-32 pb-64;
+    clip-path: polygon(0% 13%, 100% 0% ,100% 100%, 0% 87%);
+    
+    h2 {
+	    @apply pt-0 sm:pt-10 ml-5 sm:ml-10 font-extrabold tracking-wider;
+        font-size: 28px;
+
+      span {
+        @apply text-[var(--color-accent)];
+      }
+
+      @media (max-width: 640px) {
+        margin-top: 6rem;
+        width: 90%;
+       }
+    }
+
+    .flex {
+      @media (max-width: 640px) {
+       flex-direction: column;
+        margin-left: auto;
+        margin-right: auto;
+        max-width: 90%;
+        :nth-child(3) {
+        display: none;
+       }
+      }
```

Comment:
Consider using tailwind here. Check indentation.
Also of note this class comes from tailwind directly, please be careful when restyling those.